stock_picking_filter_lot_qty/models/stock_move_line.py

```diff
@@ -0,0 +1,28 @@
+from odoo import api, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    @api.onchange('product_id', 'product_uom_id')
+    def onchange_product_id(self):
+        res = super().onchange_product_id()
+        self.lot_id = False
```

Comment:
@sergiocorato I would say, as this is changing the standard behavior, it should be great to have an option on stock picking type. What do you think about ?

Reply:
I added option to filter lot by location and/or by availability.